api/python/tests/integration/test_packages.py

```diff
@@ -531,8 +535,15 @@ def test_load_into_quilt(self):
         remote_pkg.set('foo2', test_file3)
 
         with patch('time.time', return_value=1234567891), \
+             patch('quilt3.packages.DISABLE_TQDM', True), patch('quilt3.data_transfer.DISABLE_TQDM', True), \
              patch('quilt3.data_transfer.s3_transfer_config.max_request_concurrency', 1):
-            remote_pkg.push('Quilt/package', 's3://my_test_bucket/')
+            stdout = io.StringIO()
+
+            # temporarily redirect stderr to capture warnings (usually errors)
+            with redirect_stderr(stdout), patch('quilt3.packages.DISABLE_TQDM', True):
+                remote_pkg.push('Quilt/package', 's3://my_test_bucket/')
+            assert not re.search('#+', stdout.getvalue())
```

Comment:
Why not check for the ASCII block character? (I *think* it's █.) i.e. let's look for things more likely unique to tqdm.

Reply:
There isn't much output from `tqdm` except for the `0 - 100%`, `|`, and sequence of `#`.
>Why not check for the ASCII block character? (I think it's █.)

I don't understand why we need this?